pkgs/development/python-modules/swspotify/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, python3 }:
```

the individual packages also need to be passed as inputs.
```suggestion
{ stdenv, lib, buildPythonPackage, fetchPypi, reqests, flask-cors, dbus-python }:
```